releases/preprod/app.yml

```diff
@@ -7,9 +7,9 @@ metadata:
   annotations:
     fluxcd.io/automated: "true"
     filter.fluxcd.io/appealReplyServiceApi: semver:~1.1.0
-    filter.fluxcd.io/appealsServiceApi: semver:~1.3.0
-    filter.fluxcd.io/documentServiceApi: semver:~1.0.0
-    filter.fluxcd.io/formsWebApp: semver:~1.25.2
+    filter.fluxcd.io/appealsServiceApi: semver:~1.7.0
+    filter.fluxcd.io/documentServiceApi: semver:~1.1.0
+    filter.fluxcd.io/formsWebApp: semver:~1.26.1
```

Comment:
Change to `~1.26.0` - this will track anything in the `1.26.x` range thanks to the `~`